Content.Server/Nuke/NukeSystem.cs

```diff
@@ -363,7 +363,7 @@ public void DisarmBomb(EntityUid uid, NukeComponent? component = null)
             var stationUid = _stationSystem.GetOwningStation(uid);
             if (stationUid != null)
             {
-                _alertLevel.SetLevel(stationUid.Value, component.AlertLevelOnDeactivate, true, true, true);
+                _alertLevel.SetLevel(stationUid.Value, component.AlertLevelOnDeactivate, true, true, true, true);
```

Comment:
This is going to break letting users change the alert level after the alert is over.